pkg/kudoctl/cmd/install/install_test.go

```diff
@@ -26,6 +27,16 @@ func TestInstallCmd(t *testing.T) {
 	cmdWrongDirKubeConfigFlag := &cobra.Command{}
 	cmdWrongDirKubeConfigFlag.Flags().StringVar(&vars.KubeConfigPath, "kubeconfig", "", "Usage")
 	vars.KubeConfigPath = "/tmp"
+	index := []byte(`apiVersion: v1
```

Comment:
It would be really good practice here to use the functions directly, and have them take an io.ReadWriteCloser or some other interface. That way, we can use a mock/virtual filesystem instead of doing this for real. Our cmd function would then be responsible for converting a filename to an io.ReadWriteCloser.

This would also get side effects out of unit tests (though we could preserve it for e2e, and turn this into an e2e test).

I wouldn't change for now, but happy to pair through it next week!